server/auth/gmail.py

```diff
@@ -0,0 +1,53 @@
+# [START gmail_quickstart]
+from __future__ import print_function
+import pickle
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/gmail.readonly']
```

Comment:
If we want to be able to send mail, would we want a scope which would let us send mail on the user's behalf? (https://www.googleapis.com/auth/gmail.compose)
--